core/orchestrator.go

```diff
@@ -847,7 +859,28 @@ func (rtm *RemoteTranscoderManager) Manage(stream net.Transcoder_RegisterTransco
 	}
 }
 
-func (rtm *RemoteTranscoderManager) selectTranscoder() *RemoteTranscoder {
+func removeFromRemoteTranscoders(rt *RemoteTranscoder, remoteTranscoders []*RemoteTranscoder) []*RemoteTranscoder {
```

Comment:
Let's add a unit test for this. 

let's make sure to tackle following cases:
- Array length is 0 
- T to remove is the head
- T to remove is the tail
- T to remove is somewhere in the middle

Reply:
I did add the tests [here](https://github.com/livepeer/go-livepeer/pull/1849/files#diff-9734c79b7138aa52372c8c9714be7aba54b219fb23099883e1533f44d9f79385R390), although I tested with only two Ts, let me add some more and test the cases you mentioned.

Reply:
Added the change, let me know if you have any other suggestions.